docs/getting-started/install/index.md

```diff
@@ -16,23 +21,51 @@ $ brew tap shopify/shopify
 $ brew install shopify-cli
 ```
 
-### apt (Debian, Ubuntu)
+**RubyGems**
+
+See the RubyGems section below for further details.
```

Comment:
Maybe we should add a link to the right section so users don't need to look for it themselves.

Reply:
+1!